### PR TITLE
Store a single geometry inside Wkt

### PR DESCRIPTION
**Pull request by lnicola**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/master/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

r? @urschrei

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -131,28 +131,19 @@ pub struct Wkt<T>
 where
     T: WktFloat,
 {
-    pub items: Vec<Geometry<T>>,
+    pub item: Geometry<T>,
```
Comment (lnicola) [inline src/lib.rs:134]: `Wkt<T>(Geometry<T>)` might be better here? Not sure.

**Comment by michaelkirk** [comment]
I just noticed that `cargo test-all-features` doesn't seem to account for the "implicit features"  of optional dependencies.

```
$ cargo test --all-features                                                           
   Compiling wkt v0.9.2 (/Users/mkirk/src/georust/wkt)    
warning: `#[macro_use]` only has an effect on `extern crate` and modules
  --> src/conversion.rs:50:1    
   | 
50 | #[macro_use]                  
   | ^^^^^^^^^^^^                                                                     
   |
   = note: `#[warn(unused_attributes)]` on by default      
                                           
error[E0609]: no field `items` on type `Wkt<_>`       
  --> src/deserialize.rs:75:16
   |
75 |         if wkt.items.len() == 1 {
   |                ^^^^^ help: a field with a similar name exists: `item`

error[E0308]: mismatched types
  --> src/deserialize.rs:75:31
   |
62 | impl<'de, T> Visitor<'de> for GeometryVisitor<T>
   |           - this type parameter
...
75 |         if wkt.items.len() == 1 {
   |                               ^ expected type parameter `T`, found integer
   |
```

**Comment by lnicola** [comment]
I think it actually failed, but I fixed it in the meantime and did a force-push.

**Comment by michaelkirk** [comment]
Oh you're right! Sorry.

File: src/lib.rs (hunk 1 above)
Comment (michaelkirk) [reply src/lib.rs:134]: No strong opinion from me, but I have a slight preference for how you have it.

Comment (lnicola) [reply src/lib.rs:134]: I think I'd prefer renaming `Geometry` to `Wkt` or an `into_geometry()` method.

**Review by michaelkirk — APPROVED** [review]
I have one non-blocking reservation: Does this break conformance with the spec? I pretty quickly surrendered before the very long and (to me) unnavigable spec.

More pragmatically, I tested that this seems to align us with the behavior of postgis: Like the highlander, *There can be only one* top level geometry.

eg. postgis will not parse an entity like:
```
sql=# SELECT ST_GeomFromText('POINT(1 2) POINT(2 3)');
ERROR:  parse error - invalid geometry
HINT:  "POINT(1 2) POINT(2 " <-- parse error at position 19 within geometry
sql=# SELECT ST_GeomFromText('POINT(1 2), POINT(2 3)');
ERROR:  parse error - invalid geometry
HINT:  "POINT(1 2), PO" <-- parse error at position 14 within geometry
```

So, the only way to have multiple geometries in postgis, like in this PR, is to explicitly wrap with a Geometry Collection:
```
SELECT ST_GeomFromText('GEOMETRYCOLLECTION(POINT(1 2), POINT(2 3))');
```

So this LGTM, but let's see what @urschrei thinks.

**Comment by lnicola** [comment]
It's at least as conforming as the parser, since it (before my PR) parsed only one geometry out of the string.

**Comment by urschrei** [comment]
I feel the same: we're in line with PostGIS, in a less confusing way than before, so lgtm.

File: src/lib.rs (hunk 1 above)
Comment (urschrei) [reply src/lib.rs:134]: > No strong opinion from me, but I have a slight preference for how you have it.

Same, but do what you feel is best here. Where does the `into_geometry()` method come into it?

Comment (lnicola) [reply src/lib.rs:134]: > Where does the into_geometry() method come into it?

I meant `Wkt::into_geometry(self) -> Geometry`.

I'd say keep `item` for now and maybe change it in a follow-up PR if we figure out a better way.

**Comment by lnicola** [comment]
No opposition to merging and @urschrei said it's fine to merge this before #71, so let's do that.

bors r+

**Comment by michaelkirk** [comment]
Added! Can you please try again?

**Comment by lnicola** [comment]
I like how it almost takes longer to install `cargo-all-features` than it takes to run all the tests.

**Pull request merged**